File: 2022/ModoodaLive/app/src/main/java/com/creator/modoodalive/util/ProgressUtil.java
package com.creator.modoodalive.util;

import android.app.Dialog;
import android.content.Context;
import android.view.Window;

import com.creator.modoodalive.R;

/**
 * Created by skcrackers on 5/27/16.
 */
public class ProgressUtil extends Dialog {
    public ProgressUtil(Context context) {
        super(context);
        requestWindowFeature(Window.FEATURE_NO_TITLE);
        setContentView(R.layout.view_progress_util);
    }
}

File: 2025/PangPangLive/app/src/main/java/com/creator/pangpanglive/util/ProgressUtil.kt
package com.creator.pangpanglive.util

import android.app.Dialog
import android.content.Context
import android.view.Window
import com.creator.pangpanglive.R

/**
 * Created by skcrackers on 5/27/16.
 */
class ProgressUtil(context: Context?) : Dialog(context!!) {
    init {
        requestWindowFeature(Window.FEATURE_NO_TITLE)
        setContentView(R.layout.view_progress_util)
    }
}